packages/rpc/src/client.ts

```diff
@@ -1,7 +1,13 @@
+import { randomUUID } from 'node:crypto';
 import { EventEmitter } from 'node:events';
-import { setTimeout, clearTimeout } from 'timers';
-import { APIUser, OAuth2Scopes, RESTPostOAuth2ClientCredentialsResult, Routes, Snowflake } from 'discord-api-types/v10';
+import { setTimeout, clearTimeout } from 'node:timers';
+import { URLSearchParams } from 'node:url';
+import type { APIUser, OAuth2Scopes, RESTPostOAuth2ClientCredentialsResult, Snowflake } from 'discord-api-types/v10';
+import { Routes } from 'discord-api-types/v10';
 import { fetch } from 'undici';
+// eslint-disable-next-line import/extensions
+import { RPCError } from './error';
+// eslint-disable-next-line import/extensions
 import { transports } from './transports';
```

Comment:
Can't you just do this?
```suggestion
import { RPCError } from './error.js';
import { transports } from './transports/index.js';
```